static/css/zamboni/editors.styl

```diff
@@ -182,6 +182,18 @@ table.data-grid .locked .addon-locked {
     background: url(../../img/developers/editor-sprite.png?9) no-repeat top left;
     background-position: 0 -176px;
 }
+.risk_highest {
+    color: red;
+}
+.risk_high {
+    color: orange;
+}
+.risk_medium {
+    color: green;
+}
+.risk_low {
+    color: black;
+}
```

Comment:
we're usually using a dash (`-`) as a separator so I think we should do that here too.